tempesta_fw/http.c

```diff
@@ -446,9 +446,16 @@ tfw_http_conn_release(TfwConnection *conn)
  * Release resources that are not needed anymore, and keep other
  * resources that are needed while there are users of the connection.
  */
+static void tfw_http_resp_terminate(TfwHttpMsg *hm);
+
 static void
 tfw_http_conn_drop(TfwConnection *conn)
 {
+	if (conn->msg && (TFW_CONN_TYPE(conn) & Conn_Srv)) {
```

Comment:
Hm, what about POST requests? Can they be terminated by connection close as well?